test/python/circuit/test_circuit_registers.py

```diff
@@ -346,11 +346,11 @@ def test_apply_ccx_to_empty_slice(self):
         self.assertRaises(CircuitError, qc.ccx, qr[2:0], qr[4:2], qr[7:5])
 
     def test_apply_cx_to_non_register(self):
-        """test applying ccx to non-register raises"""
+        """test applying cx to non-register raises"""
         qr = QuantumRegister(10)
         cr = ClassicalRegister(10)
         qc = QuantumCircuit(qr, cr)
-        self.assertRaises(CircuitError, qc.cx, qc[0:2], qc[2:4])
+        self.assertRaises(CircuitError, qc.cx, qr[2:1], qr[3:2])
```

Comment:
I'm not even sure what this test was meant to be.  Originally it looks like it was meant to test that `qc.cx([qr[0], qr[1]], [qr[2], qr[3]])` failed, except I don't think that should have failed, since before this PR, it should have broadcast.  The test actually failed because it was passing empty lists because it used the wrong identifier, and I guess that's what you've swapped it to.  That seems reasonable, but I don't understand what it was before!

Reply:
@ewinston do you know what the purpose of this test is? 🙂 It seems to have been introduced in #1643 (ok quite a while back)